configs/google_com_proxy.v2.yaml

```diff
@@ -28,6 +28,9 @@ static_resources:
   - name: service_google
     connect_timeout: 0.25s
     type: LOGICAL_DNS
+    # uncomment the following line if your environment don't have v6 network
```

Comment:
nit: (As long as you have to merge master): 

"Uncomment the following line if your environment doesn't have v6 network access or Envoy may try to talk to a v6 endpoint which will result in a 503 error."

Reply:
thanks!